### PR TITLE
Added the ability to change the window size using env vars.

### PR DESCRIPTION
**Pull request by onodera-punpun**

### CODE REVIEW

File: browser.c [hunk 1]
```diff
@@ -1011,7 +1021,10 @@ tabbed_launch(void)
     GIOChannel *tabbed_stdout_channel;
     GError *err = NULL;
     gchar *output = NULL;
-    char *argv[] = { "tabbed", "-c", "-d", "-p", "s1", "-n", __NAME__, NULL };
+    char geom[8];
```
Comment (onodera-punpun) [inline browser.c]: I don't actually know if `8` is a mart value in this, and/or if there is a better way of converting an int to a string.

**Comment by onodera-punpun** [comment]
Woops, this has a bug, because of the tabbed geom it's no longer possible to resize the window...

File: browser.c (hunk 1 above)
Comment (vain) [inline browser.c]: `g_strdup_printf` does the job: https://developer.gnome.org/glib/stable/glib-String-Utility-Functions.html#g-strdup-printf :)

**Comment by vain** [comment]
The fact that tabbed refuses to resize the window if geometry has been set, makes it nearly impossible to implement this, doesn't it?

Guess I'll be closing this PR in a couple of days.

(Sometimes, I really like that I don't have to deal with window sizes all the time since I'm using a tiling WM.)

**Comment by onodera-punpun** [comment]
Yeah, I'll close this myself; It doesn't really work properly.

**Pull request closed without merging**